ScratchDjango/User/views.py

```diff
@@ -0,0 +1,46 @@
+import json
+
+import requests
+from django.contrib import messages
+from django.shortcuts import render
+from rest_framework import status
+from rest_framework.decorators import api_view, permission_classes
+from rest_framework.permissions import IsAuthenticated
+from rest_framework.response import Response
+
+from .forms import UserForm
+from .models import User
+
+HOST = "http://127.0.0.1:8000"
+
+
+# API Views
+@api_view(["POST"])
+def register_user(request):
+    data = request.data
+    try:
+        user = User.objects.create_user(password=data["password"], email=data["email"])
+        return Response({"Success": "User Registered"}, status=status.HTTP_200_OK)
+    except Exception as e:
```

Comment:
Can we catch actual exceptions except for broader exceptions?

Reply:
Actual exception like password validation exception or email validation exception? Email validation is set on model so any wrong email format will not be accepted and an exception message of email will be catch in e. Similarly we are validating password when creating the user in the user manager, so that exception will also be thrown in case of wrong password and will be catch in e.